test/unit/org/apache/cassandra/cql3/validation/entities/UFTest.java

```diff
@@ -1068,4 +1070,26 @@ public void testRejectInvalidLanguageOnCreation()
                                   "For more information - CASSANDRA-18252");
         }
     }
+
+    @Test
+    public void testFunctionsAreNonMonotonicAndNonDeterministicByDefault() throws Throwable
+    {
+        createFunction(KEYSPACE, "int", "CREATE FUNCTION %s (x int) " +
+                "CALLED ON NULL INPUT " +
+                "RETURNS int " +
+                "LANGUAGE java " +
+                "AS 'return x / 2;'");
+
+        UntypedResultSet functions = execute("SELECT * FROM system_schema.functions");
+
+        Assert.assertEquals(1, functions.size());
```

Comment:
This is getting all the user functions, not only the one that we have just created. It can find functions created by other test methods in this class and fail, depending on the execution order. Instead, we could do:
```java
String name = createFunction(KEYSPACE, "int", "CREATE FUNCTION %s (x int) " +
        "CALLED ON NULL INPUT " +
        "RETURNS int " +
        "LANGUAGE java " +
        "AS 'return x / 2;'");
UntypedResultSet functions = execute("SELECT * FROM system_schema.functions " +
                                     "WHERE keyspace_name=? AND function_name=?",
                                     KEYSPACE, shortFunctionName(name));
```